src/Repository/Project/PhaseRepository.php

```diff
@@ -24,4 +24,35 @@ public function getByEtudeQuery(Etude $etude)
 
         return $qb;
     }
+
+    /**
+     * @return \Doctrine\ORM\QueryBuilder
+     */
+    public function getByEtudeQueryPvId(int $pv_id, Etude $etude)
+    {
+        $qb = $this->_em->createQueryBuilder()
+            ->select('p')
+            ->from(Phase::class, 'p')
+            ->where('p.etude = :etude')
+            ->andwhere('p.procesVerbal IS NULL OR p.procesVerbal = :pv_id')
+            ->setParameter('etude', $etude)
+            ->setParameter('pv_id', $pv_id);
+
+        return $qb;
+    }
+
+    /**
+     * @return \Doctrine\ORM\QueryBuilder
+     */
+    public function getByEtudeQueryPv(Etude $etude)
```

Comment:
Et (suite du commentaire juste au dessus)

```suggestion
    public function getByEtudeWithNoPv(Etude $etude)
```

Avec éventuellement un petit commentaire au dessus de chaque pour dire pourquoi est ce que ces méthodes existent.
(Fonctionnellement parlant):

 * La première est pour retourner les phases d'un PV au cas il a déjà des phases inclues
 * La seconde est pour retourner les phases disponibles pour la création d'un PV.

Et en écrivant ce commentaire, je me rends compte que ces 2 fonctions pourraient être combinées en une seule avec l'argument `ProcesVerbal` optionnel.

Reply:
J'ai changé les noms et rajouter un commentaire pour expliquer chacune des deux